k8s/chart/templates/deployment.yaml

```
@@ -0,0 +1,134 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "sdp-operator.fullname" . }}
+  labels:
+  {{- include "sdp-operator.labels" . | nindent 4 }}
```

Helm will handle it but I think it's safer if we add a .metadata.namespace ourselves.
```suggestion
  name: {{ include "sdp-operator.fullname" . }}
  namespace: {{ .Release.Namespae }}
  labels:
  {{- include "sdp-operator.labels" . | nindent 4 }}
```